trade-finance-manager-ui/server/types/view-models/utilisation-report-reconciliation-for-report-view-model.ts

```diff
@@ -145,11 +145,26 @@ export type UtilisationTableRowViewModel = {
   };
 };
 
+export type RecordCorrectionRowViewModel = {
+  facilityId: string;
+  exporter: string;
+  baseCurrency: Currency;
+  reason: string;
+  formattedDateSent: string | undefined;
```

Comment:
Would this work?

```suggestion
  formattedDateSent?: string;
```

Reply:
this is how other dates are declared in this file so just want to keep it consistent, would rather change them all in one go

Reply:
Makes sense